docs/source/json-api/index.rst

```diff
@@ -165,7 +138,14 @@ token.  The default "header" is fine.  Under "Payload", fill in:
       }
     }
 
-Keep in mind that the value of the ``ledgerId`` payload field has to match the one passed to the sandbox with the ``--ledgerid`` argument. You can replace ``Alice`` with whatever party you want to use.
+The value of the ``ledgerId`` field has to match the ``ledgerId` of your underlying DAML Ledger.
+For the ``daml sandbox`` this corresponds to the ``--ledgerid MyLedger`` flag.
+
+.. note:: The value of ``applicationId`` can currently be anything and does not need to correspond to ``daml json-api --application-id <value>``.
```

Comment:
```suggestion
.. note:: The value of ``applicationId`` will be used for commands submitted using that token.
```

Reply:
I would just ignore the `--application-id` flag for now. I don’t see any case where it matters. I’ll open an issue for that.